dist/templates/ovnkube-identity.yaml.j2

```diff
@@ -19,8 +19,8 @@ spec:
       name: ovnkube-identity
   strategy:
     rollingUpdate:
-      maxSurge: 0
-      maxUnavailable: 1
+      maxSurge: 100%
```

Comment:
can you please explain a bit more about the upgrade scenario for 3 control-plane node case? afaiu ovnkube-identity is running on all control-plane nodes with its own leaderelection, so do we want to have 6 copies of the pod and then start deleting the old ones?